_posts/2019-03-19-Kickstarter-Trajectory-2019-Light.md

```diff
@@ -0,0 +1,296 @@
+
+We quickly learned that AWS was very advanced and had lots of components to facilitate the life of a developer.
+It was a very interesting session and made me realise that AWS was a big part of the development scene.
+We are eager to use it and learn more of what is has to offer. 
+
+<img class="image right medium" alt="OpenShift Logo" src="/img/2019-Kickstarter-Trajectory-2019-light/openshift-logo.png">
+
+As for cloud platforms, we got a very detailed explanation of how OpenShift (made by RedHat) works and what its features and options are.
+We also got a high-level explanation as to how an application in the cloud works and what the best practices are to achieve deploying your application in the cloud.
+
+Overall, it was a very interesting session for cloud-enthusiasts and we definitely want to learn more about it!
+
+## TypeScript
+After the session HTLML5, CSS3, JavaScript, [Dimitri De Kerf](/author/dimitri-de-kerf) learned us the benefits of TypeScript. 
+He told us some benefits of using TypeScript instead of JavaScript. This means that TypeScript is a wrapper around JavaScript, 
+which means it has more methods to use to develop a application which you otherwise would not be possible.
+
+Like declaring declaring a colon and a type after the variable.
+
+[Dimitri De Kerf](/author/dimitri-de-kerf) showed how to configure our project to use TypeScript and to use these features.
+He explained us that it is important to know how to use TypeScript because it is used in popular frameworks like Angular and React.
+
+
+## Angular
+
+<img class="image right" width="300" alt="Angular logo" src="/img/2019-Kickstarter-Trajectory-2019-light/angular.png">
+
+[Ryan De Gruyter](/autor/Ryan-De-Gruyter) was our teacher for today.
+He quickly introduced us to Angular, a platform that is designed to easily create browser applications built by Google. 
+The first version of Angular was AngularJS, it was very popular and used by many companies.
+Google decided to update Angular and created Angular 2 which was not welcomed by the industry at that time because it removed all the AngularJS concepts. 
+
+It took some time for the industry to adapt and see the positive stuff of Angular2: Open Source community, improved dependency injection, better performance, enc..
+Angular 2 is not the holy grail of frameworks. It still has some downsides like lots of ceremony and boilerplate thanks to the use of Angular CLI.
+
+After the information session, he showed us how easy it was to create an Angular project where we learned how to create an Angular application using small reusable pieces called components.
+
+## Frontend hands-on
+
+Today [Jan De Wilde](/autor/Jan-De-Wilde) asked if we still had question about the last Angular session. 
+Because in this session we would create an Angular application using all the techniques we learned form the previous course and he wanted to be sure we understood everything before we started.
```

Comment:
```suggestion
Because in this session we would create an Angular application using all the techniques we learned from the previous course and he wanted to be sure we understood everything before we started.
```